source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -462,6 +462,28 @@ void ESolver_KS_LCAO<TK, TR>::after_all_runners(void)
                               GlobalV::NBANDS,
                               this->p_hamilt);
     }
+
+    if (INPUT.out_mat_xc)
+    {
+        ModuleIO::write_Vxc<TK, TR>(GlobalV::NSPIN,
```

Comment:
If you write Vxc after all runners, then you can only get write_Vxc after geometry relaxation or at the end of molecular dynamics. Is this what you want?

Reply:
Yes. (Min-Ye Zhang confirmed this.)